src/manifest.json

```diff
@@ -61,7 +61,7 @@
   "optional_permissions": [
   "bookmarks",
   "downloads",
-  "browserSettings",
+  "browserSettings"
```

Comment:
Well, this extra comma after the end of the last value is not allowed by JSON, and Firefox will not load the extension if manifest.json is invalid. I'd say the JSON spec is too strict with commas.

Reply:
Yeah, it indeed, i realized it later, and i had corrected the error, and yes the error is unforgivable at the JSON level.